go/ql/src/experimental/CWE-770/DenialOfService.ql

```diff
@@ -0,0 +1,58 @@
+/**
+ * @name Denial Of Service
+ * @description slices created with the built-in make function from user-controlled sources using a
+ * maliciously large value possibly leading to a denial of service.
+ * @kind path-problem
+ * @problem.severity high
+ * @security-severity 9
```

Comment:
CI is failing because the `@problem.severity` needs to be one of `error`, `warning` or `recommendation`. See documentation [here](https://codeql.github.com/docs/writing-codeql-queries/metadata-for-codeql-queries/). It's also missing `precision`, so I'm guessing that's what you meant by `high`.
```suggestion
 * @problem.severity error
 * @security-severity 9
 * @precision high
 ```